### PR TITLE
Create Looking_for_connections.md

### PR DESCRIPTION
**Pull request by afmcdnl**
Adding a Haiku to the collection for others to enjoy.

### CODE REVIEW

File: _haikus/Looking_for_connections.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+layout: haiku
+title: Looking for connections
+author: Andrew
+---
+
+Clouds are light and white<br>
+Devices we look to connect<br>?
```
Comment (ltagliaferri) [inline _haikus/Looking_for_connections.md]: This line is one syllable over, and the question mark should be before the `<br>` tag.

A suggestion:

```
Devices look to connect?<br>
```

Comment (afmcdnl) [reply _haikus/Looking_for_connections.md]: Thanks for the suggestion!!

**Comment by afmcdnl** [comment]
Sorry, I'm new to Github and doing pull requests, I hope that the steps that I took work and it isn't too confusing.

**Comment by ltagliaferri** [comment]
Awesome, thank you for updating this pull request and for making the changes! Glad you took the time to resolve and welcome to GitHub and open source 😄

**Pull request merged**